### ISSUE TITLE
Update nvm to 1.1.5 breaks nvm

### ISSUE THREAD

**Issue by u9520107** [lukesampson/scoop#1546]
It seems the commit that added the persistence broke the installation. I used the 1.1.5 manifest prior to the persistence commit, and the installation works fine.

I'll try to test some more later.

**Comment by pepetopo** [comment]
Same here. Upgraded to NMV 1.1.5 and tried upgrading to Node 8.0.0. The installation goes smoothly, but upon trying to actually enable the new version, I get this error message:
```
C:\Git\vagrant-ansible-lemp\sites>nvm use 8.0.0
exec: "C:\\Users\\elias\\scoop\\persist\\nvm\\nodejs\\elevate.cmd": file does not exist:
```
Downgrading to older version of Node won't work after that either, and reinstallation does'nt solve the issue

**Comment by r15ch13** [comment]
Should be solved with #1547
